tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -188,42 +197,22 @@ describe(__filename, () => {
   });
 
   it('renders the license info without a link if the url is null', () => {
-    const addon = createInternalAddon({
-      ...fakeAddon,
-      current_version: {
-        ...fakeAddon.current_version,
-        license: { name: 'justText', url: null },
-      },
-    });
-    const root = render({ addon });
+    store.dispatch(_loadVersions({ license: { name: 'justText', url: null } }));
+    const root = render({});
     expect(root.find('.AddonMoreInfo-license-link')).toHaveLength(0);
 
     const link = root.find('.AddonMoreInfo-license-name');
     expect(link.children()).toIncludeText('justText');
   });
 
-  it('does not render any license info if the license is null', () => {
-    const addon = createInternalAddon({
```

Comment:
what happens if the license is null? Is that case covered still?

Reply:
I think when I wrote this patch I had updated the type definition so that license could never be null, which is why I removed this test, but then, during the review cycle I made license allowed to be null, so you're right, we need this test back. I have brought it back.